modules/social_features/social_node/social_node.module

```diff
@@ -41,3 +42,26 @@ function social_node_social_core_title(): array {
 function social_node_social_tagging_type(): string {
   return 'node';
 }
+
+/**
+ * Implements hook_node_access_records_alter().
```

Comment:
@nechai Reason of this alter is not really clear, could you add more explanation here and in the PR?

Reply:
@ribel This realm allows anonymous to see content created in group. In our case, anonymous can see content with "public" visibility only. So, we should prevent adding this realm to the content with other then "public" visibility to avoid access issues.